superduperdb/backends/sqlalchemy/metadata.py

```diff
@@ -27,10 +27,9 @@ def as_dict(self):
 class QueryID(Base):  # type: ignore[valid-type, misc]
     __tablename__ = 'query_id_table'
 
-    query_id = Column(Integer, primary_key=True, autoincrement=False)
+    query_id = Column(String, primary_key=True)
```

Comment:
This breaks due to the order of creation. Maybe a better way to do this?

Reply:
This was addressed elsewhere - by having `query_id` as string prevents book-keeping necessary on the side of the developer.